app/controllers/miq_request_controller.rb

```diff
@@ -174,13 +174,22 @@ def prov_copy
       :options        => org_req.options.except(:requester_group),
     )
 
-    prov_set_form_vars(req) # Set vars from existing request
-    # forcing submit button to stay on for copy request, setting a key in current hash so new and current are different,
-    # couldn't set this in new hash becasue that's being set by model
-    @edit[:current][:description] = _("Copy of %{description}") % {:description => org_req.description}
-    session[:changed] = true # Turn on the submit button
-    drop_breadcrumb(:name => _("Copy of %{typ} Request") % {:typ => org_req.request_type_display})
-    @in_a_form = true
+    if req.kind_of?(ServiceTemplateProvisionRequest)
+      @dialog_replace_data = req.options[:dialog].map { |key, val| {:name => key.split('dialog_').last, :value => val } }.to_json
+      @new_dialog = true
```

Comment:
@PanSpagetka I'm only seeing this instance variable getting assigned `true` on this place, but it's never reset to `false`. Is that OK?

Reply:
otherwise I don't have any comments.
I've tested the PR and changes work :+1: